README.md

```diff
@@ -2851,6 +2845,53 @@ country:
   employees. José Ancer provides a thoughtful
   [overview](http://siliconhillslawyer.com/2016/06/23/founder-compensation-cash-equity-liquidity/).
 
+🚧 This section currently mostly covers startups;
+what later-stage resources are available?
+
+### Typical Startup Advisor Equity Levels
+
+🄳 **Advisors** are people who have extensive or unique experience that help a company in a
```

Comment:
'that help a company' isn't right--should be "who help a company," with a comma after "experience"; or no comma and 'and who'